tasks/br/data.py

```diff
@@ -257,7 +256,9 @@ def columns(self):
                 targets_dict = {}
                 for x in denominators:
                     x = x.strip()
-                    targets_dict[cols.get(x, column_reqs[x].get(session) if x in column_reqs else None)] = 'denominator'
+                    target_type = 'universe' if col_agg in ['median', 'average'] else 'denominator'
+                    targets_dict[cols.get(x, column_reqs[x].get(session)
```

Comment:
For readability, we should change `x` for `denom` or `denominator`

Another question, what is the effect to have a target with universe?

Reply:
universes are used to denormalize average and median aggregations (it doesn't make sense to normalize an average for example using the area of the geometry).

It's used [here](https://github.com/CartoDB/observatory-extension/blob/6066ef028dc428d409baa6e64bf2010fa0f38064/src/pg/sql/41_observatory_augmentation.sql#L708-L724) and [here](https://github.com/CartoDB/bigmetadata/issues/326) you can find an extended explanation